src/dht/behaviour.rs

```diff
@@ -206,19 +286,69 @@ pub struct BillAttachmentFileRequest {
     pub bill_name: String,
     pub file_name: String,
 }
+/// Constructs a formatted string representing a file request for a bill attachment.
+///
+/// # Parameters
+///
+/// - `node_id`: A string slice representing the unique identifier of the node requesting the file.
+/// - `bill_name`: A string slice representing the name of the bill associated with the file.
+/// - `file_name`: A string slice representing the name of the file being requested.
+///
+/// # Returns
+///
+/// A `String` formatted as:
+/// ```text
+/// {node_id}_{BILL_ATTACHMENT_PREFIX}_{bill_name}_{file_name}
 
 pub fn file_request_for_bill_attachment(node_id: &str, bill_name: &str, file_name: &str) -> String {
     format!("{node_id}_{BILL_ATTACHMENT_PREFIX}_{bill_name}_{file_name}")
 }
-
+/// Constructs a formatted string representing a file request for a bill.
+/// # Parameters
+///
+/// - `node_id`: A string slice representing the unique identifier of the node requesting the file.
+/// - `bill_name`: A string slice representing the name of the bill associated with the file.
+///
+/// # Returns
+///
+/// A `String` formatted as:
+/// ```text
+/// {node_id}_{BILL_PREFIX}_{bill_name}
+///
 pub fn file_request_for_bill(node_id: &str, bill_name: &str) -> String {
     format!("{node_id}_{BILL_PREFIX}_{bill_name}")
 }
-
+/// Constructs a formatted string representing a file request for bill keys.
+/// # Parameters
+///
+/// - `node_id`: A string slice representing the unique identifier of the node requesting the file.
+/// - `bill_name`: A string slice representing the name of the bill associated with the keys.
+///
+/// # Returns
+///
+/// A `String` formatted as:
+/// ```text
+/// {node_id}_{KEY_PREFIX}_{bill_name}
+/// ```
```

Comment:
in 2 previous methods we don't have this line. why its here?

Reply:
we don't need them, looking at the function, since what it's does is basic formatting